### ISSUE TITLE
Score Text too small

### ISSUE THREAD

**Issue by lggauci** [YanikaZerafa/IMTAssignment2016#11]
It seems that when playing the game the score text is frankly small and difficult to see.

**Comment by YanikaZerafa** [comment]
Yes, I have to fix the text size and also the font. By fixing the typeface of the text, it would also match the rest of the UI in the game.
